react-native-ts/src/OfflineModeButton.tsx

```diff
@@ -0,0 +1,45 @@
+import React, {useEffect} from 'react';
+import {Pressable, StyleSheet} from 'react-native';
+import {Icon} from 'react-native-elements';
+import RealmContext from './RealmContext';
+
+const {useRealm} = RealmContext;
+
+export function OfflineModeButton() {
+  const realm = useRealm();
+
+  const [offlineMode, setOfflineMode] = React.useState(
+    realm.syncSession?.state === 'inactive',
+  );
+  // The signOut function calls the logOut function on the currently
```

Comment:
Is this comment a leftover from something?

Reply:
Yes, i'll clean this up.  Good catch